project-fob/Models/Fob.cs

```diff
@@ -1,37 +1,69 @@
-﻿using project_fob.Data;
+﻿using Microsoft.EntityFrameworkCore;
+using project_fob.Data;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 
 
-namespace project_fob.Models 
+namespace project_fob.Models
 {
-    public class Fob 
+    public class Fob
     {
         [Key]
         public int Id { get; set; }
-        public int AttendeeCount { get; set; }
-        public int FobCount { get; set; }
+        
+        //This variable and code is mainly here because of past values are bein loaded into the project.
+        public int AttendeeCount
+        {
+            get
+            {
+                if (Meeting != null)
+                { return -1; }
+                else { return -11; }
+            }
+            set { }
+        }
+        public int FobCount { get { return Fobbed.Count; } set { } }
+
+
         public DateTime TopicStartTime { get; set; }
-        public Meeting Meeting { get; set; } 
-        public List<Attendee> fobbed { get; set; } = new List<Attendee>();
+        public Meeting Meeting { get; set; }
+        public List<Attendee> Fobbed { get; set; } = new List<Attendee>();
 
 
         public Fob() { }
-        public Fob(Meeting meeting) 
-        { 
+        public Fob(Meeting meeting)
+        {
             // maybe check if a meeting already has a fob ? also need to check that is active
             Meeting = meeting;
-            fobbed = new List<Attendee>();
-            AttendeeCount = 0;
-            FobCount = 0;
+            Fobbed = new List<Attendee>();
+            //AttendeeCount = 0;
+            //FobCount = 0;
             TopicStartTime = DateTime.Now;
         }
-        
-        public static Fob getFob(string meetingid, ApplicationDbContext db) 
+
+        public static Fob getFob(string meetingid, ApplicationDbContext db)
         {
-            return db.Fob.SingleOrDefault(f => f.Meeting.MeetingId == meetingid);
+            return db.Fob.Include(x=> x.Meeting).ThenInclude(x=>x.Attendee).ThenInclude(x=>x.User).SingleOrDefault(f => f.Meeting.MeetingId == meetingid);
         }
+
+        public void AddFob(Attendee attendee)
+        {
+            if (!Fobbed.Contains(attendee))
+            {
+                Fobbed.Add(attendee);
+            }
+        }
+
+        public void RestartFobbed()
+        {
+            Fobbed.Clear();
+        }
+
+        /*public int GetAttendeeCount()
+        {
+            return Meeting.GetAttendeeCount();
+        }*/
```

Comment:
In general, you don't need to bother commenting out code. It's always in version control if you ever need it again, and it keeps the latest code cleaner by getting rid of stuff like that.